pkg/reconciler/delivery/delivery.go

```diff
@@ -40,14 +42,21 @@ const (
 // Reconciler implements controller.Reconciler
 type Reconciler struct {
 	client      clientset.Interface
-	routeLister listers.RouteLister 
+	routeLister listers.RouteLister
+	followup    func(*v1.Configuration, time.Duration)
```

Comment:
you probably want func(*v1.Configuration, time.Duration) as a named type.

Reply:
fixed

Reply:
I don't think I see it.

Reply:
It's in delivery.go, under the definition of Reconciler struct